python/nav/web/auth.py

```diff
@@ -17,7 +17,23 @@
 Contains web authentication functionality for NAV.
 """
 import logging
+try:
+    # Python 3.6+
+    import secrets
 
+    def fake_password(length):
+        return secrets.token_urlsafe(length)
+
+except ImportError:
+    from random import choice
+    import string
+
+    def fake_password(length):
+        S = string.ascii_letters + string.punctuation + string.digits
```

Comment:
I'm guessing you're already getting PyLint violations for invalid variable name here. `S` is both uppercase and too short...

Reply:
Fixed in """Renamed "S" to "symbols""". "S" is the mathematically correct varname to use there though.